packages/astro/CHANGELOG.md

```diff
@@ -35,6 +35,24 @@
     },
   });
   ```
+- [#7220](https://github.com/withastro/astro/pull/7220) [`459b5bd05`](https://github.com/withastro/astro/commit/459b5bd05f562238f7250520efe3cf0fa156bb45) Thanks [@ematipico](https://github.com/ematipico)! - Shipped a new SSR build configuration mode: `split`.
+
+  The Astro hook `astro:build:ssr` now receives a new option in their payload, called `entryPoints`.
```

Comment:
Tiny nit following how changesets usually format these:
```suggestion

- [#7220](https://github.com/withastro/astro/pull/7220) [`459b5bd05`](https://github.com/withastro/astro/commit/459b5bd05f562238f7250520efe3cf0fa156bb45) Thanks [@ematipico](https://github.com/ematipico)! - The Astro hook `astro:build:ssr` now receives a new option in their payload, called `entryPoints`.
```

1. Prepend a new line
2. Made `The Astro hook ...` the first changesets message instead.